src/renderer/wallet/create/password/password-util.js

```diff
@@ -11,6 +11,7 @@ export const handlePassword = (event, state) => {
 		4: 'Strong'
 	};
 	const newPassword = event.target.value;
+	console.log(newPassword);
```

Comment:
stray log

Reply:
will remove in a different pull request, almost ready